specs/linux_file_system_hierarchy.md

```diff
@@ -34,6 +34,10 @@ Some directories like `/var/spool/` are not covered,
 even though it might make sense to include them
 in the structure of an actually deployed OS.
 
+Some directories are described for compatibility with current Linux distributions,
+but their use is not recommended.
+The subsections that describe those directories are marked with ⚠️.
```

Comment:
I would add a separate heading for the section describing directories that are not recommended to make the separation stronger. Maybe you can send a follow-up for that?